src/tools/crossgen/crossgen.cpp

```diff
@@ -835,12 +833,6 @@ int _cdecl wmain(int argc, __in_ecount(argc) WCHAR **argv)
     // Are we compiling mscorlib.dll? 
     bool fCompilingMscorlib = StringEndsWith((LPWSTR)pwzFilename, CoreLibName_IL_W);
```

Comment:
`fCompilingMscorlib` looks like it's not useful anymore. It's passed through to `ComputeTPAListFromPlatformAssembliesPath` and then on to `PopulateTPAList` neither of which do anything useful with it.